Src/IronPython.Modules/_codecs.cs

```diff
@@ -366,8 +294,13 @@ public class EncodingMap {
             );
         }
 
-        public static object readbuffer_encode(string input) {
-            throw PythonOps.NotImplementedError("readbuffer_encode");
+        public static object readbuffer_encode(string input, string errors=null) {
+            return PythonTuple.MakeTuple(input, input.Length);
+        }
+
+        public static object readbuffer_encode(IPythonBufferable buffer, string errors=null) {
```

Comment:
Seems like an odd overload, Are you sure `IPythonBufferable` is the type you want?